Makefile

```diff
@@ -21,24 +21,30 @@ prereqs: ## Test if prerequisites are met, and installing missing dependencies
 	GOFLAGS="" go install github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}
 
 .PHONY: build
-build: prepare lint
+build: prepare lint ## Build the binary and run Lint
 	@go build -o $(OUTPUT)
 	cp -a ./oc/. ./$(DIST_DIR)
 	cp -a ./res/. ./$(DIST_DIR)/network-observability-cli-resources
 
 .PHONY: image
-image:
+image: ## Build the docker image
 	$(OCI_BIN) build -t network-observability-cli .
 
 .PHONY: lint
 lint: prereqs ## Lint code
 	@echo "### Linting code"
 	golangci-lint run ./...
+	@echo "### Run shellcheck against bash scripts"
+	find . -name '*.sh' | xargs shellcheck
```

Comment:
Is there a way to install `shellcheck` automatically in the `prereqs` target ?

Reply:
https://developer.ibm.com/blogs/shellcheck-and-you-should-too/

Reply:
no this a pkg u need to install on ur host and it depends on the os

Reply:
the article above suggest:
```
  docker pull koalaman/shellcheck:latest
  docker run -v "$PWD:/mnt" koalaman/shellcheck "$1"
```

The advantage of this is that `shellcheck` will not be a requirement for devs / CI

Else we need to add that in the README

Reply:
updated readme its not worth spinning up docker atm for single pkg dependency